numpy/linalg/tests/test_linalg.py

```diff
@@ -856,6 +856,22 @@ class TestPinvHermitian(PinvHermitianCases):
     pass
 
 
+def test_pinv_rtol_arg():
+    a = np.array([[1, 2, 3], [4, 1, 1], [2, 3, 1]])
+
+    with suppress_warnings() as sup:
```

Comment:
Since the warning is intensional, I'd do
```
with pytest.warns(PendingDeprecationWarning, match="In the future `rcond`"):
```

Reply:
As the warning got removed it's not needed.